### PR TITLE
Add note about conflicting settings in YAML file

### PR DESCRIPTION
**Pull request by Loquacity**
https://github.com/SUSE/spacewalk/issues/6210
https://bugzilla.novell.com/show_bug.cgi?id=1111521

### CODE REVIEW

File: adoc/retail_chap_admin.adoc [hunk 1]
```diff
@@ -110,12 +110,13 @@ If you need to change your configuration, you can edit the YAML file at any time
 Before terminals can be deployed, ensure you have prepared a saltboot based OS image.
 For how to build OS images see xref:advanced_topics_image_management.adoc#at.images.kiwi[OS image building]
 
-
 Each terminal requires a specific hardware type, which contains information about the product name and terminal manufacturer.
 However, the {susemgr} database does not have this information to begin with.
 In order to tell {susemgr} what image to deploy on each terminal, you can set hardware type groups.
 After you have created your new hardware type group, you can apply the [systemitem]``saltboot-formula`` to the group and configure it for your environment.
 
+After you have registered new terminals, always check the {susemgr} {webui} to ensure your terminal has connected successfully to the branch server, and you haven't connected directly to the {susemgr} server by mistake.
```
Comment (keichwa) [inline adoc/retail_chap_admin.adoc]: another "haven't" ;)

File: adoc/retail_chap_admin.adoc [hunk 2]
```diff
@@ -203,6 +204,15 @@ If you are working in an environment with many terminals, you might find it easi
 In order to do this, you will need to have your infrastructure planned out ahead of time, including the IP addresses, hostnames, and domain names of branch servers and terminals.
 You will also require the hardware (MAC) addresses of each terminal.
 
+[NOTE]
+====
+The settings specified in the configuration file can not always be successfully applied.
```
Comment (keichwa) [inline adoc/retail_chap_admin.adoc]: cannot (one word?)

File: adoc/retail_chap_admin.adoc (hunk 1 above)
Comment (Loquacity) [reply adoc/retail_chap_admin.adoc]: gah!

File: adoc/retail_chap_admin.adoc (hunk 2 above)
Comment (Loquacity) [reply adoc/retail_chap_admin.adoc]: yep, IBM agrees with you.

**Pull request merged**